hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueResource.kt

```diff
@@ -12,13 +13,15 @@ import org.springframework.web.server.ResponseStatusException
 
 @RestController
 @RequestMapping("/queue-admin")
-class HmppsQueueResource(private val hmppsQueueService: HmppsQueueService) {
+class HmppsQueueResource(private val hmppsQueueService: HmppsQueueService, private val hmppsAsyncQueueService: HmppsAsyncQueueService) {
```

Comment:
The main goal of this PR is to bring the non-reactive API up to date with the reactive API by handling asynchronous SQS clients.